Update project-recommendations.md (#5738)
fishtown-analytics/docs.getdbt.com: website/docs/docs/collaborate/project-recommendations.md

description: "dbt Explorer provides recommendations that you can take to improve the quality of your dbt project."
---
 

> dbt Explorer provides recommendations about your project from the `dbt_project_evaluator` [package](https://hub.getdbt.com/dbt-labs/dbt_project_evaluator/latest/) using metadata from the Discovery API. 

Reviewer: Hiya @mirnawong1 

This reads really well IMO. A possible suggestion and non-blocker - could Discovery API be hyperlinked to the Discovery API page? (https://docs.getdbt.com/docs/dbt-cloud-apis/discovery-api)

Kind Regards
Natalie

Author: good shout! yes will add!

> dbt Explorer provides recommendations about your project from the `dbt_project_evaluator` [package](https://hub.getdbt.com/dbt-labs/dbt_project_evaluator/latest/) using metadata from the Discovery API. 

Author: ```suggestion
dbt Explorer provides recommendations about your project from the `dbt_project_evaluator` [package](https://hub.getdbt.com/dbt-labs/dbt_project_evaluator/latest/) using metadata from the [Discovery API](/docs/dbt-cloud-apis/discovery-api). 
```

Reviewer: Hiya @mirnawong1 

For some reason, I couldn't add a suggestion to this.
For the sentence, **For seamless and consistent experience, recommendations are pre-defined and don't import customizations applied to your package or project** flowed nicely and read really well.

Kind Regards

- Explorer also offers a global view, showing all the recommendations across the project for easy sorting and summarizing.
- These recommendations provide insight into how you can create a better-documented, better-tested, and better-built dbt project, leading to less confusion and more trust.

> - The recommendations are pre-defined and don't import any customizations applied to your package or project.

Author: ```suggestion
- For seamless and consistent experience, recommendations are pre-defined, and don't import any customizations applied to your package or project.
```

> - For seamless and consistent experience, recommendations are pre-defined and don't import customizations applied to your package or project.

Author: ```suggestion
- For a seamless and consistent experience, recommendations uses ` dbt_project_evaluator`'s pre-defined settings and don't import customizations applied to your package or project.
```

Reviewer: Hiya @mirnawong1, I think the previous sentence was really effective and well explained. It flowed really nicely.
```suggestion
- For seamless and consistent experience, recommendations are pre-defined and don't import customizations applied to your package or project..
```

Author: Thanks @nataliefiann ! Do  you m we a the sentence that doesn't mention project evaluator? I see what you mean and i wanted to clarify or be more explicit in the exact predefined a settings that recommendation uses. I thought previously it wasn't as clear

> - For seamless and consistent experience, recommendations are pre-defined, and don't import any customizations applied to your package or project.

Author: ```suggestion
- For seamless and consistent experience, recommendations are pre-defined and don't import customizations applied to your package or project.
```

## The Recommendations tab

> Models, sources, and exposures each also have a Recommendations tab on their resource details page, with the specific recommendations that correspond to that resource:

Author: ```suggestion
Models, sources, and exposures each also have a **Recommendations** tab on their resource details page, with the specific recommendations that correspond to that resource:
```
